tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-cov
+cryptography
```

Comment:
The proxy tester does require the cryptography module in order to do certificate generation